docs/customizing-templates.rst

```diff
@@ -33,6 +33,22 @@ title for any of the fields by using the ``scaffold.fields`` configuration
         // The rest of the fields to display here
     ]);
 
+Adding Controller Actions to utilize Crud Actions
+-------------------------------------------------
+
+It's easy to add an action to a controller that makes use of another
+*CrudView* action. This uses the template provided by the add action.
+
+.. code-block:: php
+
+public function login() {
```

Comment:
Naming the method "register" would be a more realistic example.

Reply:
Good catch!